src/data/biomes/end.ts

```diff
@@ -112,4 +112,4 @@ const terrainPool = {
   [TerrainType.NONE]: 1,
 };
 
-export const endBiome = new Biome(BiomeId.END, pokemonPool, trainerPool, 0, weatherPool, terrainPool, "end");
+export const endBiome = new Biome(BiomeId.END, pokemonPool, trainerPool, 0, weatherPool, terrainPool, "end", 0);
```

Comment:
```suggestion
export const endBiome = new Biome(BiomeId.END, pokemonPool, trainerPool, 0, weatherPool, terrainPool, "end", 42.6);
```
Based on some testing (no professional methods used), this value lets the bgm loop smoothly.

Reply:
Oh wtf there was no loop point for the End biome music before?

Reply:
Yeah, end was on 0 before. I'll take 42.6 as the value since I'm also not a professional music person